symfony/plugins/orangehrmAttendancePlugin/Api/EmployeeAttendanceRecordAPI.php

```diff
@@ -63,15 +69,77 @@ class EmployeeAttendanceRecordAPI extends Endpoint implements CrudEndpoint
      */
     public function getAll(): EndpointResult
     {
```

Comment:
This is needed to be finalized. But for code reusing purpose i added this fragment and this is for employee attendance record view break  down list.